backend/src/utils/post.ts

```diff
@@ -0,0 +1,6 @@
+// TODO: enhance accuracy by taking into account HTML tags (only look at text and disregard HTML)
+export const readTime = (text: string): number => {
+  const numWords = text.split(' ').length
+
+  return Math.round(numWords / 200)
+}
```

Comment:
Is there any reason to not put this in the `utils` file that has all the other stuff around posts?